xformers/components/attention/csrc/cuda/mem_eff_attention/attention_forward_generic.cu

```diff
@@ -319,19 +320,20 @@ struct AttentionKernel {
               beta = alpha / m_prime (renormalize the output when the max
           changes) source is the current output
         */
+        int col = blockN * Mma::Shape::kN;
         OutputTileIterator output_tile_it(
             typename OutputTileIterator::Params{(int32_t)p.head_dim_value},
-            p.output_ptr + query_start() * p.head_dim_value,
+            p.output_ptr + query_start() * p.head_dim_value + col,
             typename OutputTileIterator::TensorCoord{
-                p.num_queries - query_start(), p.head_dim_value},
+                p.num_queries - query_start(), p.head_dim_value - col},
             thread_id());
         OutputTileIterator source_tile_it(
             typename OutputTileIterator::Params{(int32_t)p.head_dim_value},
-            p.output_ptr + query_start() * p.head_dim_value,
+            p.output_ptr + query_start() * p.head_dim_value + col,
             typename OutputTileIterator::TensorCoord{
-                p.num_queries - query_start(), p.head_dim_value},
+                p.num_queries - query_start(), p.head_dim_value - col},
```

Comment:
Note to self: this was a bug before, and the code wouldn't work properly for K > 128